e2e/internal/e2e/profile.go

```diff
@@ -112,6 +116,16 @@ var Profiles = map[string]Profile{
 		apptainerOption:   "--userns",
 		optionForCommands: []string{"shell", "exec", "run", "test", "instance start"},
 	},
+	overlayProfile: {
+		name:              "Overlay",
+		privileged:        false,
+		hostUID:           origUID,
+		containerUID:      origUID,
+		defaultCwd:        "",
+		requirementsFn:    overlayRequirements,
+		apptainerOption:   "",
```

Comment:
There should be a `"--userns"` apptainerOption.

Reply:
> This is only one piece of #412. There's still testing needed for using squashfuse and fuse2fs. It's fine to do this in multiple PRs but then don't say in the PR description that it fixes #412.

sorry, my bad, after adding tests for overlay, I forgot the description of this issue. will add relative tests asap.

Reply:
Not sure about the details, but I thought we did not need an unprivileged user namespace to use fuse-overlayfs?